lib/connection.c

```diff
@@ -1344,4 +1344,17 @@ int64 get_retry_timeout(SF_CONNECT *sf)
 int8 get_login_retry_count(SF_CONNECT *sf)
 {
   return (int8)get_less_one(sf->retry_on_connect_count, sf->retry_count);
+}
+
+sf_bool end_with(char* a, char* b)
+{
+    if (strlen(b) > strlen(a))
+    {
+        return SF_BOOLEAN_FALSE;
+    }
+
+    char as[1024];
```

Comment:
Although it's not performance critical we don't need a string copy here. Having a pointer set to `a + (strlen(a) - strlen(b))` would be better.

Reply:
Done.